.circleci/settings.py

```diff
@@ -0,0 +1,117 @@
+"""
+This script imports all of the required environment variables. If you are
+running it locally, update the .env file with the environment variables only
+available on your ci playform and run source.
+
+Usage: source .env && python3.6 initialize.py
+"""
+
+import os
+
+# PREFIX is the prefix of the name of the review apps created and deleted
+# by these scripts. PREFIX is used by `delete.py` to determine which apps
+# to delete after inactivity. So, it's important to use a prefix that won't
+# be used by Dash developers when creating apps manually, otherwise this
+# script may delete those apps!
+PREFIX = "review-app-"
+
+# LAST_UPDATE is the allowed amount of time before review apps are
+# deleted from the server.
+LAST_UPDATE = {"minutes": 1}
+
+# TRUNK_BRANCHNAME is the name of your repository's "main" or "master"
+# branch.
+TRUNK_BRANCHNAME = "main"
+
+# BRANCHNAME is the name of the branch you will initialize your Dash apps
```

Comment:
"you" is confusing here because it implies a user action, whereas this script is being run from CI. So maybe something like "BRANCHNAME is the name of the current branch that this script is run against. This is typically an environment variable provided by your CI platform. For example, on CircleCI this is `CIRCLE_BRANCH`."